.pre-commit-config.yaml

```diff
@@ -45,6 +45,15 @@ repos:
             | docs/\w+\.md
           )$
 
+  - repo: https://github.com/adamchainz/blacken-docs
+    rev: 1.19.0
+    hooks:
+      - id: blacken-docs
+        args: ["--line-length", "130"]
```

Comment:
Can you put this config somewhere else so you can run the command outside pre-commit?

Reply:
I don't know if this is possible (and if not, it's not a blocker), but if it is, I like the idea.

Reply:
blacken-docs does not support persistent configuration, only a very small number of CLI flags that it just forwards on to Black. We could put this configuration in our repo-wide pyproject.toml's `tool.black` configuration -- but I don't feel like I have context on why we already _have_ a pre-existing Black configuration. I think it's possibly not necessary after @calumy switched `check_docs_formatted.py` so that it uses Ruff rather than Black. (Thanks again for that PR @calumy!)

If that Black configuration section is now unnecessary, then I think I can just rework it to make it red-knot blacken-docs specific.

Reply:
Actually, it seems like even if you have a persistent black configuration in a pyproject.toml file, it's ignored by blacken-docs (because blacken-docs runs black "as a library" rather than running it as a subprocess). So it seems like there's no way to have a persistent configuration for blacken-docs at all.